bin/make/frank-plugin.sh

```diff
@@ -95,6 +101,7 @@ xcrun xcodebuild build \
   ARCHS="i386 x86_64" \
   VALID_ARCHS="i386 x86_64" \
   ONLY_ACTIVE_ARCH=NO \
+  EFFECTIVE_PLATFORM_NAME="-iphonesimulator" \
```

Comment:
Got it - How did you figure out that this was the missing arg? Seems like a very non-obvious thing to have caused that result.